### PR TITLE
Embedded site demo video youtube link

### PR DESCRIPTION
**Pull request by Lloyd-Brombach-WSU**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/testdb
 spring.datasource.username=root
-spring.datasource.password=admin
+spring.datasource.password=root
```
Comment (jacobboesch) [inline src/main/resources/application.properties]: Please change the password back.

**Review by jacobboesch — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (tia-gijo) [reply src/main/resources/application.properties]: Password change

Comment (Lloyd-Brombach-WSU) [reply src/main/resources/application.properties]: fixed

**Review by Kyle-WSU — APPROVED** [review]

**Review by TheKPham — APPROVED** [review]

File: src/main/resources/static/index.html [hunk 2]
```diff
@@ -95,48 +95,58 @@ <h3 class="ui left floated huge red header">
...
+                    <br>
+                    <br>
+                    <div class="ui container">
+                        <a href="/signup.html">
+                            <button class="ui large red button" id="Register">Register now</button>
                         </a>
-                  </div>
+                        <button class="ui large basic inverted button" id="Search" onclick="tempButton()">Search for
+                            blood
+                        </button>
```
Comment (jacobboesch) [inline src/main/resources/static/index.html]: Please remove this button as it doesn't actually work.

**Review by jacobboesch — CHANGES_REQUESTED** [review]
Just remove the broken search for blood button on the index.html page and it should be good.

File: src/main/resources/static/index.html (hunk 2 above)
Comment (Lloyd-Brombach-WSU) [reply src/main/resources/static/index.html]: Weird.. did I somehow add that? No matter  - it's gone

**Review by jacobboesch — APPROVED** [review]

**Review by tia-gijo — APPROVED** [review]

**Pull request merged**